src/adaptors/http2/http2_adaptor.c

```diff
@@ -1330,14 +1314,12 @@ ssize_t read_data_callback(nghttp2_session *session,
                 if (remaining_payload_length <= QD_HTTP2_BUFFER_SIZE) {
                 	if (length < remaining_payload_length) {
                 		bytes_to_send = length;
-                		set_buffers_to_send(stream_data, bytes_to_send);
                 		stream_data->full_payload_handled = false;
                 	}
                 	else {
                 		bytes_to_send = remaining_payload_length;
-                        set_buffers_to_send(stream_data, bytes_to_send);
                 		stream_data->full_payload_handled = true;
-                		qd_log(http2_adaptor->protocol_log_source, QD_LOG_TRACE, "[C%"PRIu64"][S%"PRId32"] read_data_callback remaining_payload_length (%zu) <= QD_HTTP2_BUFFER_SIZE(16384), bytes_to_send=%zu, stream_data->qd_buffers_to_send=%zu", conn->conn_id, stream_data->stream_id, remaining_payload_length, bytes_to_send, stream_data->qd_buffers_to_send);
+                		qd_log(http2_adaptor->protocol_log_source, QD_LOG_TRACE, "[C%"PRIu64"][S%"PRId32"] read_data_callback remaining_payload_length (%zu) <= QD_HTTP2_BUFFER_SIZE(16384), bytes_to_send=%zu", conn->conn_id, stream_data->stream_id, remaining_payload_length, bytes_to_send);
```

Comment:
should "remaining_payload_length" covert using %i instead?

Reply:
Yes, that is correct. https://github.com/apache/qpid-dispatch/commit/6d7e8c2d8bbf1645826064c3bf0074da89366071 took care of that already. I will rebase this PR